opencsp/common/lib/camera/ImageAcquisitionAbstract.py

```diff
@@ -1,8 +1,46 @@
 from abc import abstractmethod, ABC, abstractproperty
+import functools
+from typing import Callable, Optional
+
 import numpy as np
 
+import opencsp.common.lib.tool.exception_tools as et
+
 
 class ImageAcquisitionAbstract(ABC):
+    _instance: 'ImageAcquisitionAbstract' = None
+
+    @staticmethod
+    @functools.cache
+    def cam_options() -> dict[str, type['ImageAcquisitionAbstract']]:
+        """Defines camera objects to choose from (camera description, python type)"""
+        # import here to avoid circular reference
+        from opencsp.common.lib.camera.ImageAcquisition_DCAM_mono import ImageAcquisition as IA_DCAM_mono
+        from opencsp.common.lib.camera.ImageAcquisition_DCAM_color import ImageAcquisition as IA_DCAM_color
+        from opencsp.common.lib.camera.ImageAcquisition_MSMF import ImageAcquisition as IA_MSMF
+
+        cam_options: dict[str, type[ImageAcquisitionAbstract]] = {
+            'DCAM Mono': IA_DCAM_mono,
+            'DCAM Color': IA_DCAM_color,
+            'MSMF Mono': IA_MSMF,
+        }
+        return cam_options

```

Comment:
I'm not really sure what's best, just throwing this out as a potential idea. But perhaps the cam_options() method belongs in sofast_common_functions()? I'm thinking this because I believe the only purpose of this dictionary of ImageAcquisition options is for the dropdown menu in SofastGUI (and perhaps other future GUIs too) to have the correct description and associated class. Lmk what you think.

Reply:
Sounds good.